### PR TITLE
Update cloud documentation 

### PR DESCRIPTION
**Pull request by mrocklin**
cc @jacobtomlinson @jcrist for quick review

### CODE REVIEW

File: docs/source/setup/cloud.rst [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
+1.  A managed Kubernetes service and Dask's
+    :doc:`Kubernetes and Helm integraation <kubernetes-helm>`.
+2.  A managed Yarn service,
+    like `Amazon EMR <https://aws.amazon.com/emr/>`_
+    or `Google Cloud DataProc <https://cloud.google.com/dataproc/>`_
+    and :doc:`Dask-Yarn <yarn.ask.org>`_.
+
+    Specific documentation for the popular Amazon EMR service can be found
+    `here <https://yarn.dask.org/en/latest/aws-emr.html>`_
+3.  Vendor specific APIs, like Amazon ECS, and
```
Comment (jacobtomlinson) [inline docs/source/setup/cloud.rst]: Would be tempted to use the word service for consistency with other options.

```suggestion
3.  Vendor specific services, like Amazon ECS, and
```

File: docs/source/setup/cloud.rst [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
+    `Dask Cloudprovider <https://cloudprovider.dask.org/en/latest/>`_
```
Comment (jacobtomlinson) [inline docs/source/setup/cloud.rst]: `Cloudprovider`, `CloudProvider`, `Cloud Provider`?

Personally I would go for option three.

Comment (jacobtomlinson) [reply docs/source/setup/cloud.rst]: We may want to check for this in the Cloud Provider docs too.

File: docs/source/setup/cloud.rst [hunk 3]
```diff
@@ -1,21 +1,22 @@
...
+Cloud providers provide managed services,
+like Kubernetes, Yarn, or custom APIs with which Dask can connect easily.
+You may want to consider the following options:
+
+1.  A managed Kubernetes service and Dask's
+    :doc:`Kubernetes and Helm integraation <kubernetes-helm>`.
+2.  A managed Yarn service,
+    like `Amazon EMR <https://aws.amazon.com/emr/>`_
+    or `Google Cloud DataProc <https://cloud.google.com/dataproc/>`_
+    and :doc:`Dask-Yarn <yarn.ask.org>`_.
```
Comment (jrbourbeau) [inline docs/source/setup/cloud.rst]:
```suggestion
    and `Dask-Yarn <https://yarn.dask.org>`_.
```

**Comment by jcrist** [comment]
LGTM

**Pull request merged**